docs/application/native/guides/internationalization/i18n.md

```diff
@@ -76,7 +76,7 @@ The main features of the i18n API include:
   You can [format plural messages](#plural_format), with PluralFormat.
 
 > **Note**  
-> The Alphabetic Index, FieldPosition, Format, Formattable, Measure, MeasureFormat, MeasureUnit, ParsePosition, Ubidi, Ushape, and Utmscale APIs are supported from Tizen 3.0. The PluralRules, Uversion, and UCharIter APIs are supported from Tizen 4.0. The Immutable Index and the PluralFormat is supported from Tizen 5.0.
+> The Alphabetic Index, FieldPosition, Format, Formattable, Measure, MeasureFormat, MeasureUnit, ParsePosition, Ubidi, Ushape, and Utmscale APIs are supported since Tizen 3.0. The PluralRules, Uversion, and UCharIter APIs are supported since Tizen 4.0. The Immutable Index and the PluralFormat is supported from Tizen 5.0.
```

Comment:
Please change the following:
Line 808: Please remove colon from the heading
Please change as: Usage of Plural Format

Line 804, 805,806: Please can you change this 
"For predefined plural rules, see the CLDR Language Plural Rules page at http://www.unicode.org/cldr/charts/latest/supplemental/language_plural_rules.html."

to 

"For predefined plural rules, see [Language Plural Rules](http://www.unicode.org/cldr/charts/latest/supplemental/language_plural_rules.html)."